### PR TITLE
Add video to blog post and one sentence on landing page

### PR DESCRIPTION
**Pull request by akucharska**
**Description**

Changes proposed in this pull request:

- Add video to blog post and one sentence on landing page

**Related issue(s)**

### CODE REVIEW

File: src/locales/en/LandingPage.json [hunk 1]
```diff
@@ -7,6 +7,15 @@
       "Kyma is an open-source project designed natively on Kubernetes. It allows you to extend and customize cloud-based and on-premise enterprise applications in a quick and modern way, using serverless computing or microservice architecture.",
       "Kyma provides a set of cloud-native components and services required to build modern, end-to-end user experience scenarios that follow a best-practices approach to performance, scalability, efficiency, and security. Use the familiar, idiomatic language and standardized patterns that Kyma offers to extend a single enterprise application or combine existing systems to create new functionalities."
     ],
+    "blog": {
+      "paragraph": [
+        "to learn about Kyma experience by one one of the leading companies using Kyma the XXXLutz KG"
```
Comment (kazydek) [inline src/locales/en/LandingPage.json]:
```suggestion
        "to learn about Kyma experience by the XXXLutz KG, one of the leading furniture retailers."
```

File: src/components/landing/WhatIs.js [hunk 2]
```diff
@@ -8,8 +8,25 @@ const WhatIs = () => {
...
 
+  const blogLink = (
+    <div>
+      <p>
+        {blog.paragraph[0]}
+        <a
+          href={blog.link.url}
+          target="_blank"
+          rel="noopener noreferrer"
+          className="link-sig"
```
Comment (pkosiec) [inline src/components/landing/WhatIs.js]: Why `link-sig`

File: src/components/landing/WhatIs.js [hunk 3]
```diff
@@ -8,8 +8,25 @@ const WhatIs = () => {
...
   const action = landingPage[sectionPrefix].action;
+  const blog = landingPage[sectionPrefix].blog || {};
   const paragraphs = landingPage[sectionPrefix].paragraphs || [];
 
+  const blogLink = (
+    <div>
+      <p>
+        {blog.paragraph[0]}
+        <a
+          href={blog.link.url}
```
Comment (pkosiec) [inline src/components/landing/WhatIs.js]: This is internal url, so use `Link` component

File: src/components/landing/WhatIs.js [hunk 4]
```diff
@@ -8,8 +8,25 @@ const WhatIs = () => {
   const headline = landingPage[sectionPrefix].headline;
   const url = landingPage[sectionPrefix].url;
   const action = landingPage[sectionPrefix].action;
+  const blog = landingPage[sectionPrefix].blog || {};
   const paragraphs = landingPage[sectionPrefix].paragraphs || [];
 
+  const blogLink = (
```
Comment (pkosiec) [inline src/components/landing/WhatIs.js]: This is not actual link, but rather `videoLinkParagraph`

File: src/locales/en/LandingPage.json [hunk 5]
```diff
@@ -7,6 +7,15 @@
       "Kyma is an open-source project designed natively on Kubernetes. It allows you to extend and customize cloud-based and on-premise enterprise applications in a quick and modern way, using serverless computing or microservice architecture.",
       "Kyma provides a set of cloud-native components and services required to build modern, end-to-end user experience scenarios that follow a best-practices approach to performance, scalability, efficiency, and security. Use the familiar, idiomatic language and standardized patterns that Kyma offers to extend a single enterprise application or combine existing systems to create new functionalities."
     ],
+    "blog": {
+      "paragraph": [
+        "Kyma experience by XXXLutz KG, one of the leading furniture retailers."
```
Comment (kazydek) [inline src/locales/en/LandingPage.json]:
```suggestion
        "Kyma experience by XXXLutz KG, one of the leading furniture retailers"
```

**Pull request merged**